betelgeuse.py

```diff
@@ -110,10 +133,163 @@ def parse_test_results(test_results):
     return Counter([test['status'] for test in test_results])
 
 
+def add_test_case(args):
+    """Task that create or update test cases and manage manage its requirement.
+
+    This task relies on ``OBJ_CACHE`` to get the collect_only and project
+    objects.
+    """
+    path, tests = args
+    collect_only = OBJ_CACHE['collect_only']
+    project = OBJ_CACHE['project']
+    requirement = None
+    for test in tests:
+        # Expect test_case_id to be path.test_name or
+        # path.ClassName.test_name.
+        test_case_id_parts = [
+            path.replace('/', '.').replace('.py', ''),
+            test.name
+        ]
+        if test.parent_class is not None:
+            test_case_id_parts.insert(-1, test.parent_class)
+        test_case_id = '.'.join(test_case_id_parts)
+        if requirement is None:
```

Comment:
Since `requirement == None` from line 145, do we need to perform this check? I know that eventually we want to allow the user to optionally pass a `Requirement` but we currently don't do that. Perhaps check `args` for it?